.github/workflows/callable-e2e-runner.yml

```diff
@@ -39,10 +39,15 @@ jobs:
       amplifyjs_dir: ${{ matrix.integ-config.amplifyjs_dir || false }}
       browser: |
         ${{
+          fromJson(
```

Comment:
`matrix.integ-config.browser` can either be a browser (example: chrome) or browserGroup to lookup (example:     browser: minimal_browser_list).
Add necessary checks

Previously this was working because the first condition was always getting called and null was passed from the runner to the callable-test. And cypress was defaulting to chrome since no null browser was provided